indigo_optional_drivers/ccd_andor/Makefile

```diff
@@ -2,13 +2,12 @@
 # Copyright (c) 2018 Rumen G. Bogdanovski
 # All rights reserved.
 
+include ../../Makefile.inc
+
 DRIVER_NAME=indigo_ccd_andor
 
 ROOT=../..
-BUILD_ROOT=$(ROOT)/build
```

Comment:
Why did you remove those lines? This will break global make file.

Reply:
These lines were removed because I added the line:
`include ../../Makefile.inc`

`Makefile.inc` has the content of the lines removed:
```console
[root@5dac5c4dbb17 ccd_andor]# egrep 'BUILD_ROOT|BUILD_DRIVERS|BUILD_LIB|BUILD_INCLUDE' ../../Makefile.inc
BUILD_ROOT = /indigo/build
BUILD_DRIVERS = /indigo/build/drivers
BUILD_LIB = /indigo/build/lib
BUILD_INCLUDE = /indigo/build/include
```